packages/dnb-eufemia/src/core/jest/jestSetupScreenshots.js

```diff
@@ -5,66 +5,52 @@
 
 const fs = require('fs-extra')
 const path = require('path')
-const { isCI } = require('repo-utils')
 const os = require('os')
-const { setupJestScreenshot } = require('jest-screenshot')
-const { makeUniqueId } = require('../../shared/component-helper')
 const ora = require('ora')
+const { isCI } = require('repo-utils')
+const { slugify } = require('../../../src/shared/component-helper')
+const { makeUniqueId } = require('../../shared/component-helper')
 
 const log = ora()
 
-// We may use one of these: load, domcontentloaded, networkidle2
-const waitUntil = isCI ? 'load' : 'load'
-
 const config = {
   DIR: path.join(os.tmpdir(), 'jest_puppeteer_global_setup'),
   testScreenshotOnHost: 'localhost',
   testScreenshotOnPort: 8000,
-  headless: true,
-  delayDuringNonheadless: 0,
   retryTimes: isCI ? 5 : 0,
   timeout: 30e3,
-  blockFontRequest: false,
-  allowedFonts: [], // e.g. 'LiberationMono'
   pixelGrid: 8,
-  defaultViewport: {},
   pageViewport: {
     width: 1280,
     height: 1024,
-    isMobile: false,
-    hasTouch: false,
-    isLandscape: false,
-    deviceScaleFactor: 1,
   },
-  screenshotConfig: {
-    detectAntialiasing: true,
-    colorThreshold: 0.3,
-    // pixelThresholdAbsolute: 10,// when having this enabled, the Jest logs shows pixel values instead of %
-
-    // If the CI is macOS, we can have a low threshold there as well
-    // Else we opt for a slightly difference in font-rendering form setup to setup
-    pixelThresholdRelative: 0, // isCI ? 0.01 : 0
+  matchConfig: {
+    failureThreshold: isCI ? 0.01 : 0,
```

Comment:
```suggestion
    failureThreshold: 0.01,
```
This threshold seems to work on my computer running locally as well.